src/Streamly/Internal/Data/Parser/ParserD.hs

```diff
@@ -487,9 +487,22 @@ takeWhile1 predicate (Fold fstep finitial fextract) =
 --
 -- /Internal/
 --
-sliceSepBy :: -- MonadCatch m =>
+sliceSepBy :: MonadCatch m =>
     (a -> Bool) -> Parser m a b -> Parser m a b
-sliceSepBy _cond = undefined
+sliceSepBy cond (Parser pstep pinitial pextract) =
+
+    Parser step initial pextract
```

Comment:
You can put it on the same line.
```suggestion
sliceSepBy cond (Parser pstep pinitial pextract) = Parser step initial pextract
```

Reply:
It reaches column 80 because of renaming sliceSepBy to sliceSepByP